src/plugins/data/public/actions/apply_filter_action.ts

```diff
@@ -22,13 +22,19 @@ import { toMountPoint } from '../../../kibana_react/public';
 import { ActionByType, createAction, IncompatibleActionError } from '../../../ui_actions/public';
 import { getOverlays, getIndexPatterns } from '../services';
 import { applyFiltersPopover } from '../ui/apply_filters';
+import type { IEmbeddable } from '../../../embeddable/public';
 import { Filter, FilterManager, TimefilterContract, esFilters } from '..';
 
 export const ACTION_GLOBAL_APPLY_FILTER = 'ACTION_GLOBAL_APPLY_FILTER';
 
+/**
+ * Action context for {@link ACTION_GLOBAL_APPLY_FILTER}
```

Comment:
I think the @link isn't coming through on the docs: 

<img width="627" alt="Screen Shot 2020-07-09 at 9 04 35 AM" src="https://user-images.githubusercontent.com/16563603/87043622-4fcf7180-c1c3-11ea-8091-4dcd18bcbe8f.png">